benchmarks/training_loss.py

```diff
@@ -0,0 +1,100 @@
+import os
+
+import torch
+from datasets import load_dataset
+from datasets import load_metric
+from torch.optim import SGD
+from torch.utils.data import DataLoader
+from transformers import AutoModelForSequenceClassification
+from transformers import AutoTokenizer
+
+import torchdynamo
+
+torchdynamo.config.fake_tensor_propagation = False
```

Comment:
Can we turn this on? Or list down the issues that we need to fix to turn it on?

Reply:
Can we use @patch to set this, instead of setting it globally?

Reply:
> Can we turn this on? Or list down the issues that we need to fix to turn it on?

This issue has been tracked by https://github.com/pytorch/torchdynamo/issues/1076

> Can we use @patch to set this, instead of setting it globally?

I think this is both for benchmark and example, we use this to show how to enable TorchDynamo end to end. So would prefer not to add decorator on fn in case confusing users. And this is a temporary config, will remove it when we fix the above issue.